distribution.md

```diff
@@ -155,3 +155,34 @@ Where:
     * contain an array of deprecated APIs:
       * should only contain `0.x` or major versions
       * should only contain APIs that correspond to a spec release
+### Build Image
+
+The following defines how a build image SHOULD be packaged for distribution as an OCI Image. The build image is the component that provides the base image from which the build environment is constructed.
+
+The image configuration refers to the OCI Image configuration as mentioned [here](https://github.com/opencontainers/image-spec/blob/main/config.md#properties).
+
+#### Image Configuration
+
+The Build Image MUST contain the following configurations:
+
+* Image Config's `config.User` field MUST be set to the user [†](README.md#operating-system-conventions)UID/[‡](README.md#operating-system-conventions)SID with a writable home directory.
```

Comment:
Should this include a GID to match CNB_GROUP_ID?

Reply:
FWIW, I believe this was transposed from [the platform API](https://github.com/buildpacks/spec/blob/main/platform.md#build-image) which I don't see having that either.

Reply:
Hm, I guess it can be optional if the primary group is the intended group.

Reply:
Additional finding: Google and Heroku builders only use `user` (not even `uid`).

Google: `"User": "cnb"`
Heroku: `"User": "heroku"`